package.json

```diff
@@ -52,8 +54,10 @@
 		"eslint-plugin-prettier": "^2.6.1",
 		"eslint-plugin-react": "^7.9.1",
 		"husky": "^0.14.3",
+		"moment": "^2.22.2",
 		"prettier": "^1.12.0",
-		"pretty-quick": "^1.6.0"
+		"pretty-quick": "^1.6.0",
+		"react-addons-css-transition-group": "^15.6.2"
```

Comment:
@sukhrajghuman I'd prefer if we didn't need to use external packages like `moment` if we don't necessarily need it to handle our date formats? What was the problem moment.js solves that `Date()` can't do? Similar story with `react-addons-css-transition-group`?